backend/internal/service/handler/spotify/callback.go

```diff
@@ -0,0 +1,42 @@
+package spotify
+
+import (
+	"log/slog"
+	"net/http"
+	"platnm/internal/constants"
+
+	"github.com/gofiber/fiber/v2"
+	"github.com/gofiber/fiber/v2/middleware/adaptor"
+	"golang.org/x/oauth2"
+)
+
+func (h *Handler) Callback(c *fiber.Ctx) error {
```

Comment:
TODO:
- [ ] handle when state == "" so we don't query storage if not provided
- [ ] handle when the key isn't found, when sv == nil && err == nil